package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "peggy",
+  "name": "peggy-monorepo",
   "version": "1.0.0",
   "description": "Parser generator for JavaScript",
+  "private": true,
```

Comment:
wut

Reply:
yeah, the base package never gets published.

Reply:
i really hope this comment predates where you said elsewhere that you were going to back `pnpm` back out.  

![image](https://user-images.githubusercontent.com/77482/115279694-e0948f80-a0fb-11eb-91ec-ed302f35db26.png)

Reply:
Not necessarily, but I could see a compromise where what is currently in `pkg/peggy` moves down into the root directory.  I bet that would fix enough of the things that are bothering you that we could reset the discussion, so I'll do that work after lunch (MDT).

Reply:
> Also, can I ask you respectfully to remove the meme above? I don't think it's going to help this conversation resolve more quickly.

i guess

memes make it safe to show distress in a way that other people don't find to be aggressive